README.md

```diff
@@ -0,0 +1,40 @@
+# SQLite Speedtest with Emscripten
+
+This repository shows how to compile and run the speedtest1 from the SQLite
+project in the browser using different storage backends.
+
+## Getting the Source
+
+```shell
+git clone --recurse-submodules https://github.com/fivedots/sqlite-speedtest.git
+```
+
+## Running the speedtest
+To compile, navigate to the sqlite-speedtest directory and run
+
+```shell
+make all
+cd dist
+python3 -m http.server 8888
+```
+
+Then open the following link in a Chrome instance with the
+"Experimental Web Platform features" flag enabled:
+[localhost:8888/speedtest1.html](http://localhost:8888/speedtest1.html).
+
+## Results
```

Comment:
Please leave a comment here (and maybe on the HTML) about how, for now, the page has to be refreshed between benchmark executions.

Reply:
Done